src/connector/src/parser/unified/debezium.rs

```diff
@@ -158,6 +162,23 @@ pub fn parse_schema_change(
 ) -> AccessResult<SchemaChangeEnvelope> {
     let mut schema_changes = vec![];
 
+    let upstream_ddl = accessor
+        .access(&[UPSTREAM_DDL], &DataType::Varchar)?
+        .to_owned_datum()
+        .unwrap()
+        .as_utf8()
+        .to_string();
+
+    let source_id = if let Some(ScalarRefImpl::Jsonb(source_field)) =
+        accessor.access(&[SOURCE], &DataType::Jsonb)?.to_datum_ref()
+    {
+        let name: String = jsonb_access_field!(source_field, "name", string);
```

Comment:
Can we have some documentation here to explain why we can extract source id from `source.name`? I guess it is because `source.name` is derived from [`topic.prefix`](https://github.com/risingwavelabs/risingwave/blob/ba39b609cbce5b52bd59f6f6ff0a34fc4877e6e6/java/connector-node/risingwave-connector-service/src/main/resources/debezium.properties#L2) and we configure the topic prefix with source id in our codes?

Reply:
>I guess it is because source.name is derived from [topic.prefix](https://github.com/risingwavelabs/risingwave/blob/ba39b609cbce5b52bd59f6f6ff0a34fc4877e6e6/java/connector-node/risingwave-connector-service/src/main/resources/debezium.properties#L2) and we configure the topic prefix with source id in our codes?

Yes, I have refined to use `source_ctx.source_id`.